scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -819,6 +821,29 @@ def apply_filters(self, **kwargs):
                 if plat.only_tags and not set(plat.only_tags) & ts.tags:
                     instance.add_filter("Excluded tags per platform (only_tags)", Filters.PLATFORM)
 
+                snippet_args = {"snippets": ts.required_snippets}
+                snippets2 = snippets.process_snippets_inter(snippet_args, [Path(ZEPHYR_BASE)])
```

Comment:
This should include the application/test directory by default as well, as that's what `snippets.cmake` does

Reply:
Fixed